docs/data/sdks/marketing-analytics-browser/index.md

```diff
@@ -56,16 +56,35 @@ The Marketing Analytics Browser SDK has the same functionalities as the Browser
 
 ### Configuration
 
-In addition to the [basic configuration options](../typescript-browser/#configuration), the Marketing Analytics Browser SDK has options to configure web attribution and page view tracking.
+--8<-- "includes/sdk-ts-browser/shared-configurations.md"
```

Comment:
I feel it could help to mention these are shared with Browser again, since the "Usage" comment above is somewhat disconnected from this section.

```
### Configuration

Basic configuration options are the same as the standard Browser SDK.

--8<-- "includes/sdk-ts-browser/shared-configurations.md"
```